pkg/vgmanager/vgmanager_controller.go

```diff
@@ -208,11 +216,11 @@ func (r *VGReconciler) reconcile(ctx context.Context, req ctrl.Request, volumeGr
 		r.Log.Info("updated lvmd config", "VGName", volumeGroup.Name)
 	}
 
-	volGrpHostInfo, err := GetVolumeGroup(r.executor, volumeGroup.Name)
+	// volGrpHostInfo, err := GetVolumeGroup(r.executor, volumeGroup.Name)
```

Comment:
- this wasn't being used even before this PR?

Reply:
removed.